index.html

```diff
@@ -1,19 +1,110 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
     <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
-
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+  <body>
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
 
-</body>
+    <header class="header">
+      <img src="./img/karma-logo.svg" alt="" style="height: 50px" />
+      <nav class="nav-desktop">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="#"><strong>Meet Karma</strong></a>
+          </li>
+          <li class="nav-item"><a href="#">How it Works</a></li>
+          <li class="nav-item"><a href="./store.html">Store</a></li>
+          <li class="nav-item"><a href="#">Blog</a></li>
+          <li class="nav-item"><a href="#">Help</a></li>
+          <li class="nav-item"><a href="#">Login</a></li>
+        </ul>
+      </nav>
+      <button id="hamburger-menu">
+        <span></span>
+        <span></span>
+        <span></span>
+      </button>
+      <!--  -->
+<!--       <ul id="menu">
+        <li><a href="#">Contact</a></li>
+        <li><a href="#">Login</a></li>
+        <li><a href="#">Blog</a></li>
+      </ul>
+
+      <script>
+        const button = document.querySelector("#hamburger-menu");
+        const menu = document.querySelector("#menu");
+
+        button.addEventListener("click", function () {
+          menu.style.display =
+            menu.style.display === "block" ? "none" : "block";
+        });
+      </script> -->
+    </header>
+    <!-- MAIN -->
+    <main class="main">
+      <div class="main-container-up">
+        <h1>Introducing Karma</h1>
+        <h2>Bring WiFi with you, everywhere you go.</h2>
+        <button class="up-button">Learn More</button>
+      </div>
+      <!--  -->
+      <div class="main-containe-down">
+        <h1 class="down-h1">Everyone needs a little Karma.</h1>
```

Comment:
please try to use more semantic tags. instead of div tag , you can use section tag.